### PR TITLE
test(integration): Add test for large error inbox

### LINKED ISSUE #362: Increase Maximum TransactionError Message Size
256 Char Limit -> 1024 Char Limit
See MMF for more details.

- [x] Code changes
- [x] New tests,
- [x] Update existing tests

### PR DESCRIPTION
**Pull request by ZNeumann**
This test is added to verify that our agent does not have a error message max size

Closes #362

### CODE REVIEW

File: tests/integration/errors/test_large_error_message.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+// Attempt to get rid of our exception handler.
+restore_exception_handler();
```
Comment (misiekn) [inline tests/integration/errors/test_large_error_message.php:62]: Is this call to `restore_exception_handler()` needed? There aren't any exception handlers previously set with [`set_exception_handler()`](https://www.php.net/manual/en/function.set-exception-handler.php)

Comment (ZNeumann) [reply tests/integration/errors/test_large_error_message.php]: fixed

**Review by misiekn — APPROVED** [review]

**Review by mfulb — APPROVED** [review]
Sure would be nice to have a preprocessor for these tests a shame to have to duplicate that long string.

Looks fine given our limitations.

**Comment by zsistla** [comment]
Thanks for doing this :)

How many characters is the error message?
Does the full error message show up in NR1?

**Comment by ZNeumann** [comment]
> Thanks for doing this :)
> 
> How many characters is the error message? Does the full error message show up in NR1?

It's around 1400 characters and the entire message shows up in NR1

**Review by zsistla — APPROVED** [review]

**Pull request merged**